setup.cfg

```diff
@@ -26,6 +26,7 @@ python_requires = >=3.6
 install_requires =
     ruamel.yaml==0.17.20
     importlib-resources==5.4.0; python_version < "3.9"
+    jinja2
```

Comment:
Yes, technically this is in the wrong PR, however there's a command in the CircleCI config that will break without this (`mots ci`) since it imports the export command, which also imports jinja2. This is also included in pr #7 .

Reply:
For more context: that command is run from an install that's outside the dev environment. Thus, it relies on all the relevant requirements being in setup.cfg. This is really a separate bug, but it is fixed here and in the other PR.